common/djangoapps/student/models.py

```diff
@@ -251,7 +251,11 @@ class Meta:  # pylint: disable=missing-docstring
     goals = models.TextField(blank=True, null=True)
     allow_certificate = models.BooleanField(default=1)
     bio = models.TextField(blank=True, null=True)
-    has_profile_image = models.BooleanField(default=0)
+    profile_image_version = models.CharField(max_length=8, null=True)
```

Comment:
Couldn't you store the actual timestamp in an int? That seems like it might be more useful, and it is half the size. I could imagine showing the timestamp to the user (last changed two days ago, or something).